src/View/SerializedView.php

```diff
@@ -84,8 +85,23 @@ public function render($view = null, $layout = null)
         }
 
         if ($serialize !== false) {
-            return $this->_serialize($serialize);
-        } elseif ($view !== false && $this->_getViewFileName($view)) {
+            $result = $this->_serialize($serialize);
+            if ($result === false) {
+                if (Configure::read('debug') && in_array('trace', $serialize)) {
```

Comment:
Sniffing around for blacklisted keys feels like it could end us up in trouble longer term. This reminds me of when we hid all the connection related keys in debug() and had to revert many of them as they were hiding useful data.